src/backend.js

```diff
@@ -106,6 +97,17 @@ var _ = Mavo.Backend = $.Class({
 			call.searchParams.set("timestamp", Date.now());
 		}
 
+		if ($.type(req.data) === "object") {
+			if (req.method == "GET") {
+				Object.keys(req.data).map(p => call.searchParams.set(p, req.data[p]));
+				req.data = call.search;
+				call.search = "";
```

Comment:
Why is this step needed? Isn't the whole point of doing this to add it as a query string to the call URL?
If it's to avoid adding it twice, you could just clear `req.data`.